src/webui/www/private/scripts/client.js

```diff
@@ -771,17 +796,49 @@ window.addEvent('load', function() {
         if (showSearchEngine) {
             $('showSearchEngineLink').firstChild.style.opacity = '1';
             $('mainWindowTabs').removeClass('invisible');
-
-            addMainWindowTabsEventListener();
+            $('searchTabLink').removeClass('invisibleImp');
+            if (!showRssReader) {
```

Comment:
Can you extract this conditional to a named variable to better document its purpose (e.g. `const tabsInititalized = `)